examples/game_of_life/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "game_of_life"
+version = "1.0.0"
+description = "MeasurementLink example that displays Conway's Game of Life in a graph."
+authors = ["National Instruments"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+ni-measurementlink-service = {version = "^1.2.0-dev0", allow-prereleases = true}
+click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pallets/click/issues/2558
+
+[tool.poetry.group.dev.dependencies]
+mypy = ">=1.0"
+grpc-stubs = "^1.53"
+types-protobuf = "^4.21"
+# Uncomment to use prerelease dependencies.
+ni-measurementlink-service = {path = "../..", develop = true}
+
+[tool.mypy]
```

Comment:
@bkeryan I put this to try to enforce type hints in this example. However, it allowed this where **kwargs doesn't have a type hint and it doesn't check functions prepended with '_' (private methods). Is this expected or is mypy not checking things correctly?
```python
def main(verbosity: int, **kwargs) -> None:
```

Reply:
Looks like the PR checks caught these things. I must have been invoking it incorrectly locally.